EquivalenceIT fails reproducibly with a seed

```
  2> REPRODUCE WITH: gradle :core:integTest -Dtests.seed=A4648847991E5C27 -Dtests.class=org.elasticsearch.search.aggregations.EquivalenceIT -Dtests.method="testRandomRanges" -Dtests.security.manager=true -Dtests.jvms=12 -Dtests.locale=nn-NO -Dtests.timezone=Europe/Helsinki
FAILURE 5.55s J10 | EquivalenceIT.testRandomRanges <<< FAILURES!
   > Throwable #1: java.lang.AssertionError: 4 expected:<889> but was:<890>
   >    at __randomizedtesting.SeedInfo.seed([A4648847991E5C27:D00DCF66E03EFC10]:0)
   >    at org.elasticsearch.search.aggregations.EquivalenceIT.testRandomRanges(EquivalenceIT.java:186)
   >    at java.lang.Thread.run(Thread.java:745)
```

--- a/core/src/test/java/org/elasticsearch/search/aggregations/EquivalenceIT.java
+++ b/core/src/test/java/org/elasticsearch/search/aggregations/EquivalenceIT.java
@@ -103,7 +103,18 @@ public class EquivalenceIT extends ESIntegTestCase {
             }
         }
 
-        createIndex("idx");
+        prepareCreate("idx")
+                .addMapping("type", jsonBuilder()
+                        .startObject()
+                            .startObject("type")
+                                .startObject("properties")
+                                    .startObject("values")
+                                        .field("type", "double")
+                                    .endObject()
+                                .endObject()
+                            .endObject()
+                        .endObject()).execute().actionGet();
+
         for (int i = 0; i < docs.length; ++i) {
             XContentBuilder source = jsonBuilder()
                     .startObject()

--- a/core/src/test/java/org/elasticsearch/search/aggregations/EquivalenceIT.java
+++ b/core/src/test/java/org/elasticsearch/search/aggregations/EquivalenceIT.java
@@ -213,6 +213,9 @@ public class EquivalenceIT extends ESIntegTestCase {
                         .startObject()
                             .startObject("type")
                                 .startObject("properties")
+                                    .startObject("num")
+                                        .field("type", "double")
+                                        .endObject()
                                     .startObject("string_values")
                                         .field("type", "keyword")
                                         .startObject("fields")
@@ -334,7 +337,18 @@ public class EquivalenceIT extends ESIntegTestCase {
 
     // Duel between histograms and scripted terms
     public void testDuelTermsHistogram() throws Exception {
-        createIndex("idx");
+        prepareCreate("idx")
+                .addMapping("type", jsonBuilder()
+                        .startObject()
+                            .startObject("type")
+                                .startObject("properties")
+                                    .startObject("num")
+                                        .field("type", "double")
+                                    .endObject()
+                                .endObject()
+                            .endObject()
+                        .endObject()).execute().actionGet();
+
 
         final int numDocs = scaledRandomIntBetween(500, 5000);
         final int maxNumTerms = randomIntBetween(10, 2000);
@@ -394,7 +408,17 @@ public class EquivalenceIT extends ESIntegTestCase {
 
     public void testLargeNumbersOfPercentileBuckets() throws Exception {
         // test high numbers of percentile buckets to make sure paging and release work correctly
-        createIndex("idx");
+        prepareCreate("idx")
+                .addMapping("type", jsonBuilder()
+                        .startObject()
+                            .startObject("type")
+                                .startObject("properties")
+                                    .startObject("double_value")
+                                        .field("type", "double")
+                                    .endObject()
+                                .endObject()
+                            .endObject()
+                        .endObject()).execute().actionGet();
 
         final int numDocs = scaledRandomIntBetween(2500, 5000);
         logger.info("Indexing [{}] docs", numDocs);

